requirements-build.txt

```diff
@@ -0,0 +1,50 @@
+#
+# This file is autogenerated by pip-compile with python 3.8
+# To update, run:
+#
+#    pip-compile --allow-unsafe --output-file=requirements-build.txt requirements-build.in
```

Comment:
Can we have this documented somewhere in repo how to do it, or even better we have a Makefile

`make requirements` (or magic, pip-compile) and documented